src/policy/policy.h

```diff
@@ -68,6 +68,7 @@ bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
 
 namespace Policy {
     std::int32_t blockSizeAcceptLimit();
+    std::int64_t blockSigOpAcceptLimit(int32_t nBlockSize);
```

Comment:
its an int, not unsigned, so only 2GB. :)
No, I don't think this is a concern any time soon.